pkg/imager/profile/output.go

```diff
@@ -27,6 +27,11 @@ type Output struct {
 	//  * .xz - output xz archive
 	//  * .gz - output gz archive
 	OutFormat OutFormat `yaml:"outFormat"`
+	// Bootloader is the bootloader to use for the image.
+	// If not set, it defaults to dual-boot.
+	// Secureboot always uses sd-boot.
+	// ARM64 uses sd-boot for Talos >= 1.10.
+	Bootloader ImageBootloader `yaml:"bootloader"`
```

Comment:
This feels like a breaking change to me... 

As much as I hate it, I feel we might need to duplicate this in Image and ISO options :(